scipy/signal/tests/test_signaltools.py

```diff
@@ -1295,7 +1295,7 @@ def test_odd_strides(self, xp):
         # us into wrong memory if used (but it does not need to be used)
         dummy = xp.arange(10, dtype=xp.float64)
         a = dummy[5:6]
-        a.strides = 16
+        a = xp.lib.stride_tricks.as_strided(a)
```

Comment:
I don't know if using `xp` here contributes that much--pretty sure the stride trick stuff can't be part of the array API standard anyway.

Reply:
I should probably set the value to `16` though...

Reply:
`xp.asarray(np.stride_tricks.as_strided(np.asarray(a),...))) `

Reply:
Or make the test numpy-only via `@skip_xp_baxkends(np_only=True) `